modules/ROOT/pages/storage.adoc

```diff
@@ -264,9 +264,13 @@ storage:
       with_mount_unit: true
 ----
 
-The root filesystem can also be moved to LUKS. In the case of the root filesystem the LUKS device must be backed by https://coreos.github.io/ignition/operator-notes/#clevis-based-devices[clevis]. There is simplified Butane config syntax for encrypting the root filesystem; for example:
+The root filesystem can also be moved to LUKS. In that case, the LUKS device must be pinned by https://coreos.github.io/ignition/operator-notes/#clevis-based-devices[Clevis]. There are two primary pin types available: TPM2 and Tang (or a combination of those using Shamir Secret Sharing).
 
-.Moving the root filesystem to LUKS
+CAUTION: TPM2 pinning just binds encryption to the physical machine in use. Make sure to understand its threat model before choosing between TPM2 and Tang pinning. For more information, see https://github.com/latchset/clevis/blob/master/src/pins/tpm2/clevis-encrypt-tpm2.1.adoc#threat-model[this section] of the Clevis TPM2 pin documentation.
```

Comment:
This link provides little information which treat models one is protected from and which - not. I guess I'm looking for a more comprehensive guide but well. It seems in general topic is complicated and I don't find well systematized information about it.

Reply:
Yes, I agree the upstream documentation could have more details about tradeoffs. I opened https://github.com/latchset/clevis/issues/381 to track this. If a section gets added, we can update these docs to link to it.